site/content/en/docs/Guides/utilize-node-externaldns.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Utilize Node ExternalDNS"
+linkTitle: "Utilize Node ExternalDNS"
+date: 2020-12-14
+weight: 30
+description: "Use External DNS as the address of the game server."
+---
+
+{{< alpha title="Utilize Node ExternalDNS" gate="NodeExternalDNS" >}}
+
+## Description
```

Comment:
Not sure this warrants it's own specific entry into the documentation Guide section :thinking: ?

It feels to me like an extension of this faq entry: https://agones.dev/site/docs/faq/#how-are-ip-addresses-allocated-to-gameservers

Extending that would probably work fine, and if we find that that section is getting too large on review, we should have a whole section around Address resolution, rather than a section just for this particular feature.

How does that sound?

Reply:
That's exactly right.
I'll try to add documentation to the items you suggested :)